features/networking/pod.feature

```diff
@@ -190,3 +190,119 @@ Feature: Pod related networking scenarios
     Then the step should succeed
     And the output should not contain "ingress_policing_rate: 1953"
 
+  # @auther anusaxen@redhat.com
+  # @case_id OCP-23890
+  @admin
+  Scenario: A pod with or without hostnetwork cannot access the MCS port 22623 or 22624 on the master
+    Given I use the first master host
+    #Step to obtain master IP
+    And I run commands on the host:
+      | hostname -i |
+    Then the step should succeed
+    And evaluation of `@result[:response].strip` is stored in the :master_ip clipboard
+    
+    Given I select a random node's host
+    Given I have a project
+    #pod-for-ping will be a non-hostnetwork pod
+    And SCC "privileged" is added to the "system:serviceaccounts:<%= project.name %>" group
+    And I have a pod-for-ping in the project
+    And the pod named "hello-pod" becomes ready
+
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22623/master/config | -k |
+    Then the output should contain "Connection refused"
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22624/master/config | -k |
+    Then the output should contain "Connection refused"
+    
+    #hostnetwork-pod will be a hostnetwork pod
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/anuragthehatter/v3-testfiles/master/networking/hostnetwork-pod.json |
+      | n | <%= project.name %>                                                                                   |
+    Then the pod named "hostnetwork-pod" becomes ready
+    #Pods should not access the MCS port 22623 or 22624 on the master
+    When I execute on the pod:
+      | curl | -I | https://<%= cb.master_ip %>:22623/master/config | -k |
```

Comment:
just found the path is not correct, should be 'https://<%= cb.master_ip %>:22623/config/master' or https://<%= cb.master_ip %>:22623/config/worker'

Reply:
thats right, thanks for catching that. Ye manually i tested on config/worker, will re-run tests again after this change

Reply:
on behalf of @zhaozhanqi , this has been taken care of as confirmed with developer as well